timeless/restaurants/tables/views.py

```diff
@@ -0,0 +1,63 @@
+"""tables views module.
+@todo #112:30min Continue implementing list_tables(), create(), edit() and
+ delete() methods, using SQLAlchemy and Location model. In the index page it
+ should be possible to sort and filter for every column. Location management
+ page should be accessed by the Location page. Update html templates when
+ methods are implemented. Create more tests for all methods.
+ Table management pages should be accessed by the Location and Floor pages.
+"""
+import datetime
+
+from flask import (
+    Blueprint, flash, redirect, render_template, request, url_for
+)
+
+
+bp = Blueprint("table", __name__, url_prefix="/tables")
+
+
+@bp.route("/")
+def list():
+    # remove this dummy tables object and use db
+    tables = [{
+        "id": 1,
+        "name": "Test table",
+        "description": "This is a test",
+        "floor_id": 1,
+        "x": 10,
+        "y": 20,
+        "width": 12,
+        "height": 22,
+        "status": 1,
+        "max_capacity": 6,
+        "multiple": True,
+        "playstation": False,
+        "shape_id": 3,
+        "min_capacity": 4,
+        "deposit_hour": 3,
+        "created": datetime.datetime.now(),
+        "updated": datetime.datetime.now(),
+        }]
+    return render_template("restaurants/tables/list.html", tables=tables)
+
+
+@bp.route("/create", methods=("GET", "POST"))
```

Comment:
@krzyk I would prefer to register the endpoints in a central point. For instance, see how the ``/companies`` api is registered in ``timeless/__init.py`` (method ``register_api``).

Reply:
@amihaiemil It depends, both ways have their pros and cons.
@emilianodellacasa Where should we add API endpoints in view classes or like it was done in case of companies in a single (assume it will be a big) method in timeless/__init.py?

Reply:
@krzyk Let's add it in timeless/__init__.py